packages/main/src/RPA/Images.py

```diff
@@ -200,7 +203,8 @@ def crop_image(self, image, region, filename=None, save_format="PNG"):
         image.load()
 
         if filename:
-            image.save(filename, save_format)
+            # Suffix isn't created automatically here
+            image.save(filename + ".png", "PNG")
```

Comment:
should this also use pathlib?

Reply:
Yeah that's probably a more stable solution